unified_planning/model/expression.py

```diff
@@ -318,13 +318,54 @@ def Forall(
             node_type=OperatorKind.FORALL, args=expressions, payload=vars
         )
 
+    def Always(self, *expression: BoolExpression) -> "up.model.fnode.FNode":
+        if len(expression) != 1:
+            raise UPExpressionDefinitionError(
+                f"Always of expression: {str(expression)} must be created with exactly one parameter."
+            )
+        expressions = tuple(self.auto_promote(expression))
+        return self.create_node(node_type=OperatorKind.ALWAYS, args=expressions)
+
+    def Sometime(self, *expression: BoolExpression) -> "up.model.fnode.FNode":
+        if len(expression) != 1:
+            raise UPExpressionDefinitionError(
+                f"Sometime of expression: {str(expression)} must be created with exactly with one parameter."
+            )
+        expressions = tuple(self.auto_promote(expression))
+        return self.create_node(node_type=OperatorKind.SOMETIME, args=expressions)
+
+    def AtMostOnce(self, *expression: BoolExpression) -> "up.model.fnode.FNode":
+        if len(expression) != 1:
+            raise UPExpressionDefinitionError(
+                f"At-Most-Once of expression: {str(expression)} must be created with exactly with one parameter."
+            )
+        expressions = tuple(self.auto_promote(expression))
+        return self.create_node(node_type=OperatorKind.AT_MOST_ONCE, args=expressions)
+
+    def SometimeBefore(self, *expression: BoolExpression) -> "up.model.fnode.FNode":
+        if len(expression) != 2:
+            raise UPExpressionDefinitionError(
+                f"Sometime-Before of expression: {str(expression)} must be created with exactly with 2 parameters."
+            )
+        expressions = tuple(self.auto_promote(expression))
```

Comment:
```suggestion
    def SometimeBefore(self, left: BoolExpression, right: BoolExpression) -> "up.model.fnode.FNode":
        expressions = tuple(self.auto_promote(left, right))
```
Here, `left` and `right`, should be more meaningful names, depending on the given semanthic.